nin/frontend/app/styles/common.less

```diff
@@ -530,8 +530,29 @@ input[type=range]:focus::-ms-fill-upper {
   padding-bottom: 10px;
   font-family: monospace;
   z-index: 999999999999999;
-  background-color: rgba(113, 75, 75, 0.6);
-  overflow-y: scroll;
```

Comment:
Dammit I can't seem to move or delete line notes. I've commented globally as well.